packages/clerk-js/src/ui/primitives/Link.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import { useCoreClerk } from '../contexts';
```

Comment:
Primitives should not be aware of clerk utils - let's use a HOC for that.

Reply:
Good suggestion!